pipelines/build/common/build_base_file.groovy

```diff
@@ -127,7 +127,13 @@ class Builder implements Serializable {
         if (additionalBuildArgs) {
             buildArgs += ' ' + additionalBuildArgs
         }
-        def enableReproducibleCompare = isEnableReproducibleCompare(platformConfig, variant)
+
+        def configEnableReproducibleCompare = false
+        if (enableReproducibleCompare) {
```

Comment:
I'm not an expert on this code, but this seems to be using `enableReproducibleCompare` when the definition of that is being removed. Have I missed something?

Reply:
Thanks - that explanation makes it clear :-)